xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm

```diff
@@ -148,19 +151,20 @@ $xwiki.ssfx.use('uicomponents/pagination/pagination.css', true)##
     #pagination ($paginationParameters)
       <div class="form-inline">
         #if ($displayCompare)
-          <input type="submit" accesskey="c" value="$services.localization.render('core.viewers.history.compare')" class="btn btn-primary"/>
+          <input type="submit" name="displaycompare" accesskey="c" value="$services.localization.render('core.viewers.history.compare')" class="btn btn-primary" />
```

Comment:
Camel case ``displayCompare`` would be easier to read (the previous code was not following the best practice, but we can improve it).

Reply:
ok.